IntegrationTests/BTCardClient_IntegrationTests.swift

```diff
@@ -5,7 +5,7 @@ import XCTest
 class BTCardClient_IntegrationTests: XCTestCase {
 
     func testTokenizeCard_whenCardHasValidationDisabledAndCardIsInvalid_tokenizesSuccessfully() {
-        var apiClient = BTAPIClient(authorization: BTIntegrationTestsConstants.sandboxTokenizationKey)!
+        var apiClient = BTAPIClient(authorization: BTIntegrationTestsConstants.sandboxTokenizationKey)
```

Comment:
nit: I think we can make the same changes we made to the apple pay integration tests in these tests as well